### PR TITLE
bugfix/PLAYER-5121 Improve 360 video accessibility

### PR DESCRIPTION
**Pull request by KseniiaSokolovaEpam**
Accessibility for navigation buttons for video 360  was added.

Unit tests passed.

[Related PR](https://github.com/ooyala/skin-config/pull/233)

### CODE REVIEW

File: js/components/directionControlVr.jsx [hunk 1]
```diff
@@ -19,7 +20,15 @@ class DirectionControlVr extends React.Component {
    * @param {Event} event - event object
    */
   rotateVrVideo(event) {
-    const isRotated = event && (event.type === 'mousedown' || event.type === 'touchstart');
+    const charCode = event.which || event.keyCode;
+    const enterCharCode = 13;
+    const spaceCharCode = 32;
```
Comment (zelibobla) [inline js/components/directionControlVr.jsx]: Keycodes constants are here:
https://github.com/ooyala/html5-skin/blob/master/js/constants/constants.js#L252-L261

Comment (KseniiaSokolovaEpam) [reply js/components/directionControlVr.jsx]: @zelibobla oh, yes, exactly. Thank you, Anton!

File: js/components/directionControlVr.jsx [hunk 2]
```diff
@@ -32,39 +42,70 @@ class DirectionControlVr extends React.Component {
...
         onMouseOut={this.rotateVrVideo}
+        onBlur={this.rotateVrVideo}
+        onKeyDown={this.rotateVrVideo}
+        onKeyUp={this.rotateVrVideo}
+        language={language}
+        localizableStrings={localizableStrings}
+        responsiveView={responsiveView}
+        skinConfig={skinConfig}
+        controller={controller}
+        key={focusId}
```
Comment (ooyala-amohnatkin) [inline js/components/directionControlVr.jsx]: you can't be sure that it's always unique

File: js/components/directionControlVr.jsx [hunk 3]
```diff
@@ -19,7 +21,15 @@ class DirectionControlVr extends React.Component {
    * @param {Event} event - event object
    */
   rotateVrVideo(event) {
-    const isRotated = event && (event.type === 'mousedown' || event.type === 'touchstart');
+    const charCode = event.which || event.keyCode;
```
Comment (ooyala-amohnatkin) [inline js/components/directionControlVr.jsx]: is it correct to check `event.which || event.keyCode` on `onMouseDown` or `onTouchStart` events?

Comment (KseniiaSokolovaEpam) [reply js/components/directionControlVr.jsx]: 2 functions for these types of events were added(as you wanted :-) )

**Pull request merged**